member/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Member(models.Model):
+    user = models.OneToOneField(User)
+
+    # Facebook field
+    uuid = models.CharField('User ID', max_length=100)
+    email = models.EmailField('Email')
+
+    def __str__(self):
+        return self.user.get_username()
+
+    def __unicode__(self):
+        return self.user.get_username()
```

Comment:
I made the data store in customized model with a field for Django auth so as to get the  current request user by `request.user`. Is the implementation right for this purpose?